ui/src/tall_stats.tsx

```diff
@@ -0,0 +1,69 @@
+// Copyright 2020 H2O.ai, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import { Model } from 'h2o-wave'
+import React from 'react'
+import { Stat } from './stats'
+import { cards } from './layout'
+import { bond } from './ui'
+import { stylesheet } from 'typestyle'
+import { clas, cssVar } from './theme'
+
+/** Create a vertical label-value pairs collection. */
+export interface State {
+  /** The individual stats to be displayed. */
+  items: Stat[]
+}
+
+const
+  css = stylesheet({
+    card: {
+      display: 'flex',
+      flexDirection: 'column',
+      textAlign: 'center',
+      padding: 24,
+    },
+    stat: {
+      marginTop: 24,
+    },
+    value: {
+      color: cssVar('themePrimary'),
+    },
+    label: {
+      color: cssVar('neutralPrimary'),
+      marginTop: -8,
+    }
+  })
+
+export const
+  View = bond(({ name, state, changed }: Model<State>) => {
+    
+    const
+      render = () => {
+        const stats = state.items.map(({ label, value }, i) => (
+          <div key={`${i}-${label}`} className={i ? css.stat : ''}>
```

Comment:
Use flexbox spacing instead.
```suggestion
          <div key={`${i}-${label}`}>
```